src/containers/Home/dashboardHome.style.ts

```diff
@@ -0,0 +1,11 @@
+import { styled } from "@mui/material/styles";
+import MuiAccordionDetails from "@mui/material/AccordionDetails";
+
+export const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
```

Comment:
If you want, you can use the Project Details screen structure for accordion and others, it's already created, which may give us the same look and feel for entire the application, thanks